packages/@aws-cdk/cloud-assembly-schema/lib/schema.ts

```diff
@@ -0,0 +1,90 @@
+import * as fs from 'fs';
```

Comment:
I got to say that the file names should be reverse. This should be `manifest.ts` (i.e. exports the `Manifest` class) and the other is the actual schema...

Reply:
Yeah, i actually had it that way before. 👍